inet/vibe/inet/webform.d

```diff
@@ -581,10 +581,12 @@ unittest
 	bsonMap["╤╳"] = "1";
 
 	assert(urlEncode(aaMap) == "complex=%E2%95%A4%E2%95%B3%2F%3D%24%24%22%271%212%28%29%27%22&unicode=%E2%95%A4%E2%95%B3&spaces=1%202%203%204%20a%20b%20c%20d&numbers=123456789&slashes=1%2F2%2F3%2F4%2F5&equals=1%3D2%3D3%3D4%3D5%3D6%3D7&%E2%95%A4%E2%95%B3=1" ||
```

Comment:
So we have five keys here which could result in `5! = 20` different combinations?
Maybe it's better and more readable to do sth. like:

```
urlEncode(aaMap) .split("&").sort.release == "..."
```